src/script/SubmitSendWithCompose.s.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: ISC
+pragma solidity ^0.8.19;
+
+import { BaseScript } from "frax-std/BaseScript.sol";
+import { console } from "frax-std/FraxTest.sol";
+
+import { OptionsBuilder } from "@fraxfinance/layerzero-v2-upgradeable/oapp/contracts/oapp/libs/OptionsBuilder.sol";
+import { SendParam, OFTReceipt, MessagingFee, IOFT } from "@fraxfinance/layerzero-v2-upgradeable/oapp/contracts/oft/interfaces/IOFT.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract SubmitSendWithCompose is BaseScript {
+    using OptionsBuilder for bytes;
+
+    // address oft = 0x909DBdE1eBE906Af95660033e478D59EFe831fED; // Base FRAX OFT
+    address oft = 0xF010a7c8877043681D59AD125EbF575633505942; // Base frxETH OFT
+    address swapMock = 0x60356998558A466Ec51BdE7e78F3b88Bdc843c5e; // Fraxtal MockReceiver
```

Comment:
This is the `MockReceiver` contract using a different variable name here.  This specific contract was used to test bridging `wfrxETH` from a different chain to Fraxtal and atomically swapping and unwrapping to the recipient.  You can see the test tx [here](https://fraxscan.com/tx/0x3121ed5fb4bf84e017945e5d861d0f6ac59dccaed0ea360c5dc70e3b932ab562).